src/components/sponsor_card.tsx

```diff
@@ -0,0 +1,57 @@
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import Image from "next/image";
+
+type sponsor_card_props = {
```

Comment:
could we change this to an interface and rename this to `SponsorCardProps` to keep prop conventions consistent?

Reply:
👍